src/app/modules/main/node_section/view.nim

```diff
@@ -0,0 +1,150 @@
+import NimQml, chronicles, json, strutils
+import io_interface
+import ../../../core/signals/types
+
+
+QtObject:
+  type
+    View* = ref object of QObject
+      delegate: io_interface.AccessInterface
+      callResult: string
+      lastMessage*: string
+      stats*: Stats
+      peerSize: int
+      bloomBitsSet: int
+ 
+  proc delete*(self: View) =
+    self.QObject.delete
+
+  proc newView*(delegate: io_interface.AccessInterface): View =
+    new(result, delete)
+    result.QObject.setup
+    result.delegate = delegate
+    result.callResult = "Use this tool to call JSONRPC methods"
+    result.lastMessage = ""
+
+  proc load*(self: View) =
+    self.delegate.viewDidLoad()
+
+  proc callResult*(self: View): string {.slot.} =
+    result = self.callResult
+
+  proc callResultChanged*(self: View, callResult: string) {.signal.}
+
+  proc setCallResult(self: View, callResult: string) {.slot.} =
+    if self.callResult == callResult: return
+    self.callResult = callResult
+    self.callResultChanged(callResult)
+
+  proc `callResult=`*(self: View, callResult: string) = self.setCallResult(callResult)
+
+  QtProperty[string] callResult:
+    read = callResult
+    write = setCallResult
```

Comment:
@borismelnik remove `write` prop, as it's `callResult` is used only to be read on the qml side.